code/game/objects/items/twohanded.dm

```diff
@@ -1017,3 +1017,256 @@
 		C.change_view(CONFIG_GET(string/default_view))
 		user.client.pixel_x = 0
 		user.client.pixel_y = 0
+
+/obj/item/twohanded/electrostaff
+	icon = 'icons/obj/estaff.dmi'
+	icon_state = "electrostaff_3"
+	item_state = "electrostaff_3"
+	lefthand_file = 'icons/mob/inhands/weapons/estaff_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/estaff_righthand.dmi'
+	name = "riot suppression electrostaff"
+	desc = "A large quarterstaff, with massive silver electrodes mounted at the end."
+	force = 10
+	damtype = BRUTE
+	w_class = WEIGHT_CLASS_GIGANTIC
```

Comment:
Huge wieght class is better for this, as gigantic things are not ment to be picked up

Reply:
aight bet